src/components/error-summary/index.md.njk

```diff
@@ -9,25 +9,26 @@ layout: layout-pane.njk
 
 {% from "_example.njk" import example %}
 
+Use an error summary when a there is a validation error. The error summary uses the error message that explains what went wrong and how to fix it.
+
 {{ example({group: 'components', item: 'error-summary', example: 'default', html: true, nunjucks: true, open: false}) }}
 
 ## When to use this component
 
-You must always show an error summary when there is a validation error with the information that a user has entered, even if there’s only one.
+Always show an error summary when there is a validation error, even if there’s only one.
 
 ## How it works
 
 You must:
 
+- add 'Error: ' to the beginning of the `<title>` so screen readers read it out as soon as possible
```

Comment:
Change:

add 'Error: '

to:

add 'Error:'

Reply:
I think it needs to be add 'Error: ' to make sure people know there is a space after the colon.

Reply:
Ah OK - it just looks strange on the preview as the second ' is curling forwards like an opening quote. I don't know if there's any way to fix that.

Reply:
I think the prime is interpreted as a left single quote because there is a space before it. You could force a right single quote in the markdown file. See the character codes and ways to do it on http://smartquotesforsmartpeople.com/

Reply:
Updated to use the left and right single quotes.